### PR TITLE
Add conductor 1.2.1 (initial release)

### PR DESCRIPTION
**Pull request by keith**
// @vitorgalvao

### CODE REVIEW

File: Casks/conductor.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'conductor' do
+  version '1.2.1'
+  sha256 '0df00414d64474093aeefd9f7b81fd80bb8a98da2b713164e498df6cd0e60cca'
+
+  url 'https://github.com/keith/conductor/releases/download/1.2.1/Conductor.app.zip'
```
Comment (vitorgalvao) [inline Casks/conductor.rb]: You can add a `appcast 'https://github.com/keith/conductor/releases.atom'`. Line underneath this one, and we should be able to auto-update, in the future.

Comment (keith) [inline Casks/conductor.rb]: Wow! I didn't know that was possible. Awesome!

Comment (keith) [inline Casks/conductor.rb]: Updated!

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as e3e07636de451683f45784e5d33f127918f443e5 to keep commit history cleaner. Your contribution is still credited to you.

**Comment by keith** [comment]
Thanks @vitorgalvao !

**Pull request closed without merging**